### DISCUSSION TITLE
Не отключается WEB интерфейс на 88 порту

### DISCUSSION THREAD

**Discussion by D-Daene** [Anonym-tsk/nfqws-keenetic#119] [Q&A] [answered]
Обновился до последней версии 2.5.11.

WEB интерфейс переехал и работает на 90 порту,
но также продолжает работать на старом 88 порту.
Как остановить WEB интерфейс на 88 порту?

Перезапуск не помогает:
/opt/etc/init.d/S80lighttpd restart

Удаление и установка также не помогает.

**Comment by Anonym-tsk** [comment, marked as answer]
После перезагрузки роутера отключится. Это особенность работы uhttpd
